src/main/java/helper/command/AddCommand.java

```diff
@@ -0,0 +1,68 @@
+package helper.command;
+
+import helper.DukeException;
+import helper.Storage;
+import helper.TaskList;
+import helper.Ui;
+import task.Deadline;
+import task.Event;
+import task.Todo;
+
+import java.time.LocalDate;
+
+/**
+ * Commands for adds (todo..., deadline..., event...)
+ */
+public class AddCommand extends Command {
+
+    private final String whichAdd;
+    private final String addString;
+
+    public AddCommand(String whichAdd, String description) {
+        this.whichAdd = whichAdd;
+        this.addString = description;
+    }
+
+    /**
+     * Executes the parsed command
+     * @param tasks
+     * @param ui
+     * @param storage
+     * @throws DukeException
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        switch (whichAdd) {
+            case "todo":
```

Comment:
Indentation should be inline with the switch according to our coding standard. I still don't know why they are doing that...